ansible/roles/cleaner/tasks/main.yml

```diff
@@ -160,29 +63,3 @@
   include_tasks: scenarios/srm-toolchain.yml
   when: install_srm_toolchain == true
   tags: srm_toolchain
-
-- name: clean all configuration and log files
-  file:
-    path: "{{ item }}"
-    state: absent
-    force: yes
-  with_items:
-    - "{{ opensds_config_dir }}"
-    - "{{ opensds_log_dir }}"
-    - "{{ opensds_certs_dir }}"
-    - "{{ delfin_config_dir }}"
-  ignore_errors: yes
-
-- name: clean all created work directories
```

Comment:
Is this being taken care of elsewhere?
These two blocks clean the directories created.

Reply:
Yes, cleanup is separated project wise